src/crawler/index.js

```diff
@@ -34,11 +34,17 @@ crawlerConfigs.forEach((crawler) => {
 });
 
 // hardcoded loop
-function runLoop() {
-    crawlers.forEach(async (crawler) => {
+crawlers.forEach(async (crawler) => {
+    
+    async function runLoop() {
         await crawler.crawl();
-    });
-};
-
-runLoop();
-let timeout = setInterval(runLoop, 5 * 60 * 1000);
\ No newline at end of file
+    }
+    runLoop()
+    if (crawler.config.update.includes("d")) {
```

Comment:
Should you use an array let timeouts = []; and then push a new timeout for each crawler to the timeouts array? This would be better since timeouts would not be "lost" to the code.